dotcom-rendering/src/components/FootballMatchList.tsx

```diff
@@ -437,6 +440,13 @@ export const FootballMatchList = ({
 	const [days, setDays] = useState(initialDays);
 	const [isError, setIsError] = useState<boolean>(false);
 
+	const [jsLoaded, setJsLoaded] = useState(false);
+
+	useEffect(() => {
+		// This will only run after JS has loaded on the client
+		setJsLoaded(true);
+	}, []);
```

Comment:
So sorry @marjisound, I've just noticed we have a [`useHydrated` hook](https://github.com/guardian/dotcom-rendering/blame/main/dotcom-rendering/src/lib/useHydrated.ts) that does the same thing. Would you be able to use it here?

Reply:
Ah thanks for letting me know, just changed it to use the `useHydrated` hook